homeassistant/components/wemo/switch.py

```diff
@@ -81,45 +92,50 @@ def extra_state_attributes(self):
             attr["on_total_time"] = WemoSwitch.as_uptime(
                 self.wemo.insight_params.get("ontotal", 0)
             )
-            attr["power_threshold_w"] = (
-                convert(self.wemo.insight_params.get("powerthreshold"), float, 0.0)
-                / 1000.0
+            threshold = convert(
+                self.wemo.insight_params.get("powerthreshold"), float, 0.0
             )
+            assert isinstance(threshold, float)
+            attr["power_threshold_w"] = threshold / 1000.0
 
         if isinstance(self.wemo, CoffeeMaker):
             attr[ATTR_COFFEMAKER_MODE] = self.wemo.mode
 
         return attr
 
     @staticmethod
-    def as_uptime(_seconds):
+    def as_uptime(_seconds: int) -> str:
         """Format seconds into uptime string in the format: 00d 00h 00m 00s."""
         uptime = datetime(1, 1, 1) + timedelta(seconds=_seconds)
         return "{:0>2d}d {:0>2d}h {:0>2d}m {:0>2d}s".format(
             uptime.day - 1, uptime.hour, uptime.minute, uptime.second
         )
 
     @property
-    def current_power_w(self):
+    def current_power_w(self) -> float | None:
         """Return the current power usage in W."""
         if isinstance(self.wemo, Insight):
```

Comment:
Would save some indent if you reversed the condition and `return None` in the indent, then outdent below

Reply:
Thanks! Rearranged this method and the one below.